listenbrainz/mbid_mapping/mapping/mapping_test/mapping_test.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import config
+
+from mapping.search import search
+
+
+def _read_test_data(filename):
+    data = []
+    with open(filename, newline='') as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='"')
+        for row in reader:
+            if not row:
+                if config.USE_MINIMAL_DATASET:
+                    break
+                else:
+                    continue
+
+            data.append(row)
+
+    return data
+
+
+def test_mapping():
+    ''' This test will run as many test as there are in the CSV file '''
+
+    data = _read_test_data("mapping/mapping_test/mapping_test_cases.csv")
+
+    passed = 0
+    failed = 0
+
+    for row in data:
+        hits = search(row[1] + " " + row[0])
+        if row[2] != hits[0]['release_mbid']:
+            print("Q %-30s %-30s %-25 %s" % (row[0][:29], "", row[1][:29], row[2]))
+            print("H %-30s %-30s %-25 %s" % (hits[0]['recording_name'][:29], hits[0]['release_name'][:29],
+                  hits[0]['artist_credit_name'][:29], hits[0]['release_mbid']))
+            print
+            failed += 1
```

Comment:
do you want this to report the total number of passes and failures at the end of running, or could you use pytest here and just `assert` it? (It'll quit after the first failure)

Reply:
I fail to see, (ha!) why it would quit after the first failure. Previously testing was more complicated and setting up pytest was a pita, but I can do this in a future PR. "good enough" for this moment.